haystack/components/evaluators/llm_evaluator.py

```diff
@@ -208,11 +211,14 @@ def run(self, **inputs) -> Dict[str, Any]:
                 results.append(None)
                 errors += 1
 
+            if self.api == "openai" and "meta" in result:
+                metadata = result["meta"]
+
         if errors > 0:
             msg = f"LLM evaluator failed for {errors} out of {len(list_of_input_names_to_values)} inputs."
             warn(msg)
 
-        return {"results": results}
+        return {"results": results, "metadata": metadata}
```

Comment:
for consistency with the rest of the code base, I would call it `meta` instead of `metadata`.

Reply:
👍🏽